www/css/style.css

```diff
@@ -14,6 +19,20 @@ div.spinner {
   height: 85px !important;
 }
 
+.space-between {
+    display: flex;
+    /* justify-content: space-between; */
+    align-items: center;
+    /* align-items: flex-end; */
+}
+
+.speaker-photo {
+    /* width: 40%; */
```

Comment:
I tried fixing the image being cut off. Do you have any idea how to fix that? Do we need to just upload a circle photo in the database? 
<img width="374" alt="screen shot 2018-02-21 at 11 33 55 pm" src="https://user-images.githubusercontent.com/10505327/36523639-d6328c8c-175f-11e8-8a09-84ea6741e612.png">